src/main/resources/story/reddit/reddit-search-and-open-post.story

```diff
@@ -0,0 +1,24 @@
+Description: Test that create user account
+
+Lifecycle:
+Examples:
+{transformer=FROM_LANDSCAPE}
+|first-post         |By.xpath((//*[@data-testid="post-container" and .//@data-click-id="body"])[1])|
+
+Scenario: Login in
+Given I am on a page with the URL 'https://www.reddit.com/'
+When I wait until element located `By.xpath(//a[@role="button"][text()="Log In"])` appears
+When I click on element located `By.xpath(//a[@role="button"][text()="Log In"])`
+When I switch to frame located `By.xpath(//*[contains(@src,"reddit.com/login")])`
+When I wait until element located `By.xpath(//main//input[@id="loginUsername"])` appears
+When I enter `New-Objective3067` in field located `By.xpath(//main//input[@id="loginUsername"])`
```

Comment:
It is a good practice to put your test values into Examples section

Reply:
Fixed, test values are set in Examples.